### ISSUE TITLE
BlogListViewController: crash on setSelectedBlog:animated

### ISSUE THREAD

**Issue by kurzee** [wordpress-mobile/WordPress-iOS#6781]
Looks like a split-view related crash.

cc @frosty 

### Version

WPiOS 6.8+

### Log

```
Fatal Exception: NSInvalidArgumentException
0  CoreFoundation                 0x18ac8e1c0 __exceptionPreprocess
1  libobjc.A.dylib                0x1896c855c objc_exception_throw
2  UIKit                          0x190b746d0 -[UINavigationController loadView]
3  UIKit                          0x190b73d40 -[UINavigationController pushViewController:animated:]
4  WordPress                      0x100144c50 -[BlogListViewController setSelectedBlog:animated:] (BlogListViewController.m:725)
5  WordPress                      0x100144a70 -[BlogListViewController setSelectedBlog:] (BlogListViewController.m:708)
6  WordPress                      0x1001449e8 -[BlogListViewController tableView:didSelectRowAtIndexPath:] (BlogListViewController.m:703)
7  WordPress                      0x100143d74 __53-[BlogListViewController newWordPressComBlogCreated:]_block_invoke (BlogListViewController.m:582)
8  WordPress                      0x10023b97c thunk (PushAuthenticationServiceRemote.swift)
9  WordPress                      0x10023b97c thunk (PushAuthenticationServiceRemote.swift)
10 WordPress                      0x1003546d8 specialized UITableView.(flashRowAtIndexPath(IndexPath, scrollPosition : UITableViewScrollPosition, flashLength : Double, completion : () -> ()?) -> ()).(closure #1) (UITableView+Helpers.swift)
11 WordPress                      0x100354654 partial apply for UITableView.(flashRowAtIndexPath(IndexPath, scrollPosition : UITableViewScrollPosition, flashLength : Double, completion : () -> ()?) -> ()).(closure #1) (UITableView+Helpers.swift)
12 libdispatch.dylib              0x189b191c0 _dispatch_client_callout
13 libdispatch.dylib              0x189b26008 _dispatch_continuation_pop
14 libdispatch.dylib              0x189b32648 _dispatch_source_latch_and_call
15 libdispatch.dylib              0x189b1b164 _dispatch_source_invoke
16 libdispatch.dylib              0x189b1dbc0 _dispatch_main_queue_callback_4CF
17 CoreFoundation                 0x18ac3bf2c __CFRUNLOOP_IS_SERVICING_THE_MAIN_DISPATCH_QUEUE__
18 CoreFoundation                 0x18ac39b18 __CFRunLoopRun
19 CoreFoundation                 0x18ab68048 CFRunLoopRunSpecific
20 GraphicsServices               0x18c5ee198 GSEventRunModal
21 UIKit                          0x190b4d2fc -[UIApplication _run]
22 UIKit                          0x190b48034 UIApplicationMain
23 WordPress                      0x1000772c0 main (main.m:5)
24 libdispatch.dylib              0x189b4c5b8 (Missing)
```

**Comment by hypest** [comment]
Seems related to this crash report: http://crashes.to/s/03076805667

**Comment by jleandroperez** [comment]
Closing as dupe of #7284. Should be fixed by #8406.

Thanks everyone!
